galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/src/main/java/dev/galasa/http/internal/HttpClientImpl.java

```diff
@@ -68,6 +68,7 @@
 
 import com.google.gson.JsonObject;
 
+import dev.galasa.common.SSLTLSContextName;
```

Comment:
It's not a context name you are adding here. It's a SSLTLSContextNameSelector or something similar.
I would expect a ContextName class to contain a name of some sort...
This is a class which just does the selection of the name given an environment. Can you change the name pls ?